diff --git a/cocoapods-patch-20230121-69200-ep4h6l/Swifter/XCode/Sources/WebSockets.swift b/Pods/Swifter/XCode/Sources/WebSockets.swift
index a2d46e3..04f7923 100644
--- a/cocoapods-patch-20230121-69200-ep4h6l/Swifter/XCode/Sources/WebSockets.swift
+++ b/Pods/Swifter/XCode/Sources/WebSockets.swift
@@ -32,7 +32,7 @@ public func websocket(
             return .badRequest(.text("Invalid value of 'Sec-Websocket-Key' header: \(request.headers["sec-websocket-key"] ?? "unknown")"))
         }
         let protocolSessionClosure: ((Socket) -> Void) = { socket in
-            let session = WebSocketSession(socket)
+            let session = WebSocketSession(socket, request)
             var fragmentedOpCode = WebSocketSession.OpCode.close
             var payload = [UInt8]() // Used for fragmented frames.
 
@@ -161,9 +161,11 @@ public class WebSocketSession: Hashable, Equatable {
     }
 
     public let socket: Socket
+    public let request: HttpRequest?
 
-    public init(_ socket: Socket) {
+    public init(_ socket: Socket, _ request: HttpRequest? = nil) {
         self.socket = socket
+        self.request = request
     }
 
     deinit {
