diff --git a/MusicKit.Events.d.ts b/MusicKit.Events.d.ts
index 427bf2dd50b7c88651b4f3e77dde0d6592da20db..76d8adb271684234be9911875351f6c4265c4a0c 100755
--- a/MusicKit.Events.d.ts
+++ b/MusicKit.Events.d.ts
@@ -19,6 +19,10 @@ declare namespace MusicKit {
          * A notification name indicating MusicKit JS is loaded.
          */
         loaded: unknown;
+        /**
+         * autoplayEnabled has been changed
+         */
+        autoplayEnabledDidChange: unknown;
         /**
          * A notification name indicating the player has obtained enough data for
          * playback to start.
