diff --git a/node_modules/@types/musickit-js/MusicKit.Player.d.ts b/node_modules/@types/musickit-js/MusicKit.Player.d.ts
index 2319a40..293de1c 100755
--- a/node_modules/@types/musickit-js/MusicKit.Player.d.ts
+++ b/node_modules/@types/musickit-js/MusicKit.Player.d.ts
@@ -2,7 +2,7 @@ declare namespace MusicKit {
     /**
      * The playback states of the music player.
      */
-    enum PlaybackState {
+    enum PlaybackStates {
         NONE,
         LOADING,
         PLAYING,
